tests/opentracing_instrumentation/test_sqlalchemy.py

```diff
@@ -42,9 +42,10 @@ def test_db(tracer, session):
     session.commit()
 
     spans = tracer.recorder.get_spans()
-    assert len(spans) == 3
+    assert len(spans) == 4
```

Comment:
how is this changing without any code changes?

Reply:
[Changes in SQLAlchemy](https://github.com/sqlalchemy/sqlalchemy/commit/be985bd41ecd0eeff90c1a817167a34eb02aeed1#diff-79c290956ad49825a085d3073170ca03) cause performing one additional `PRAGMA` operation which leads to creation of a span.
That changes [are part](https://github.com/sqlalchemy/sqlalchemy/compare/rel_1_3_6...rel_1_3_7) of [SQLAlchemy 1.3.7 release](https://pypi.org/project/SQLAlchemy/1.3.7/).

I realized that I forgot to update the requirements, so I've updated them now.